pom.xml

```diff
@@ -20,6 +20,7 @@
     <jackson.version>2.8.9</jackson.version>
     <jenkins.version>2.46.3</jenkins.version> <!-- Should be kept in sync with Dockerfile FROM statement-->
     <javadoc.exec.goal>javadoc-no-fork</javadoc.exec.goal> <!-- stop initialize phase plugins executing twice -->
+    <frontend-version>1.6</frontend-version>
```

Comment:
Vas ist das?

Reply:
that fixes the maven front end plugin to the version of 1.6 which has more sensible failure mode when running npm/scripts that return failure (to avoid that stupid thing of last week you found). 

This is already on master BTW (so we don't get a new front end plugin by surprise in future).

Reply:
...but why here?